src/components/Badge/Badge.tsx

```diff
@@ -47,6 +49,9 @@ function Badge({ color = 'brand', children, size, dot }: Props) {
   if (size === 'large') {
     classes.push(__styles.size.large)
   }
+  if(overlayClasses){
```

Comment:
change prop name to be `className` ?

other than that, should be fine.

Reply:
above changes are pushed.

Reply:
nice one!